### DISCUSSION TITLE
Is it possible to use openscap to create an ansible playbook directly from the xccdf file provided by the DISA Stig?

### DISCUSSION THREAD

**Discussion by glenc2004** [OpenSCAP/openscap#2044] [Q&A] [answered]
Hello. I'm wondering if it's possible to create an Ansible playbook using openscap from the xccdf file provided by the DISA stig. When I try and use it, I don't get any output. When I turn on verbose mode I see openscap says there is "no fix element was found for rule ......".

So can it be done?

Thanks

GlenC

**Comment by glenc2004** [comment, marked as answer]
I answered my own question. I see the ssg* files contain the ansible code for playbook creation while the DISA STIG does not!

GlenC

**Reply by ggbecker** [reply]
DISA also provides similar ansible content that you can find here: https://public.cyber.mil/stigs/supplemental-automation-content/

But I have no idea about the quality and usefulness of it.
